zipline/history/history.py

```diff
@@ -278,6 +285,8 @@ def __init__(self, bar_count, frequency, field, ffill,
         # The frequency at which the data is sampled.
         self.frequency = frequency
         # The field, e.g. 'price', 'volume', etc.
+        if not isinstance(field, Hashable):
```

Comment:
Looking at this again, I think we should just assert that this is in HistoryContainer.VALID_FIELDS.  We might have to move that somewhere shared to fix a circular dependency.